css/main.css

```diff
@@ -99,3 +101,509 @@ body{
+  margin: 0;
+  font-family: 'Mulish', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 36px;
+  line-height: 48px;
+  color: #000; 
+}
+.text__title__section-4::before{
+  position: absolute;
+  content: "";
+  height: 2px;
+  width: 56px;
+  background: #000;
+  top: 126px;
+  left: 0;
+}
+.text__title__section-4::after{
+  position: absolute;
+  content: "";
+  height: 2px;
+  width: 56px;
+  background-color: #000;
+  top: 126px;
+  left: 0;
+} 
+.text__p-section-4{
+  width: 420px;
+  height: 120px;
+  margin: 63px 0 0 0;
+  font-family: 'Mulish', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 30px;
+  color: #7d7987;
+}
+.button-section-4{
+  height: 56px;
+  width: 200px;
+  margin-top: 40px;
+  padding: 0;
+  border-radius: 55px;
+  border: 1.4px solid #458ff6;
+  font-family: 'Mulish', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 60px;
+  color: #458ff6;
+  background: #fff;
+}
+.image_illustration3{
+  margin: 229px 164px 0 0;
+}
+.elements-3{
+  position: absolute;
+  top: -10px;
+  left: 1514px;
+  height: 115px;
+  width: 86px;
+  opacity: 0.3;
+}
+.vector__elements-3{
+  width: 10px;
+  height: 10px;
+  margin: 10px 0 0 14px;
+}
+.button-section-3{
+  height: 56px;
+  width: 200px;
+  margin-top: 70px;
+  padding: 0;
+  border-radius: 55px;
+  border: 1.4px solid #458ff6;
+  font-family: 'Mulish', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 60px;
+  color: #458ff6;
+  background: #fff;
+}
+.button-section-3:hover{
+  color: #e608ee;
+  background: #461172;
+}
+.button-section-3:focus{
+  color: #0b8000;
+  outline: none;
+  border: 2px solid #eeb808;
+}
+.button-section-3:active{
+  color: #0673f0;
+  background: #ae0ae0;
+  outline: none; 
+}
+.button-section-4{
+  height: 56px;
+  width: 200px;
+  margin-top: 70px;
+  padding: 0;
+  border-radius: 55px;
+  border: 1.4px solid #458ff6;
+  font-family: 'Mulish', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 60px;
+  color: #458ff6;
+  background: #fff;
+}
+.button-section-4:hover{
+  color: #e608ee;
+  background: #461172;
+}
+.button-section-4:focus{
+  color: #0b8000;
+  outline: none;
+  border: 2px solid #eeb808;
+}
+.button-section-4:active{
+  color: #0673f0;
+  background: #ae0ae0;
+  outline: none; 
+}
+@font-face{
+  font-family: 'icomoon';
+  src: url('../fonts/icomoon.woff?43ej29') format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+.icon{
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'icomoon' !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+.icon-arrow-download:after{
+  content: "\e900";
```

Comment:
There is should be only content property. For rest of properties use anther class.